doc/source/security/encrypting-tables.rst

```diff
@@ -0,0 +1,41 @@
+.. _encrypting-tables:
+
+=========================================================
+Encrypting File-Per-Tablespace Tables
+=========================================================
+
+When you define the variable ``innodb_file_per_table`` in the my.cnf file, InnoDB uses a tablespace file for each InnoDB table and creates and stores the table data and the indexes in a single data file. In this tablespace configuration, each table is stored in an .ibd file.
```

Comment:
When you define the variable ``innodb_file_per_table`` in the my.cnf file - this is also not needed as innodb_file_per_table is ON by default. Maybe just explain that innodb_file_per_table is ON by default which means Innodb uses a tablespace file for each (...) unless you explicitly specify innodb_file_per_table = OFF in my.cnf. Something like this ?

Reply:
Fixed.